backend/municipal-services/ingestion-service/app/processor/boundary_data_processor.py

```
@@ -0,0 +1,265 @@
+            boundary_df = self.data_loader.get_boundary_data()
+            logger.info(boundary_df.head(2))
+        else:
+            logger.warning("Data loader is not compatible")
+            return pd.DataFrame()
+
+        # Run all validators
+        has_error = False
+        for validator in self.validators:
+            boundary_df = validator.validate(boundary_df)
+            if (boundary_df["status"] == "fail").any():
+                has_error = True
```

Ruff (0.8.2)

Local variable `has_error` is assigned to but never used

Remove assignment to unused variable `has_error`

(F841)